integreat_cms/cms/utils/matomo_api_manager.py

```diff
@@ -347,15 +363,23 @@ def get_visits_per_language(self, start_date, end_date, period):
                 # The dataset for offline downloads
                 + [
                     {
-                        "label": _("Offline Downloads"),
+                        "label": _("Offline Accesses"),
```

Comment:
```suggestion
                        "label": _("Offline Downloads"),
```
Maybe offline is the only use case where we should avoid talking about "accesses", because we don't know how often the users actually access the app? Not sure though, would also be fine for me to use the same wording for all columns.

Reply:
Well, I would stay consistent over all columns. Especially because we summarize the offline downloads/accesses in the total accesses column as well.

Reply:
Alright, makes sense! :+1: